docs/content/1.documentation/2.headers/3.crossOriginEmbedderPolicy.md

```diff
@@ -53,7 +53,7 @@ Cross-Origin-Embedder-Policy: require-corp
 The `crossOriginEmbedderPolicy` header can be configured with following values.
 
 ```ts
-crossOriginEmbedderPolicy: 'unsafe-none' | 'require-corp' | false;
+crossOriginEmbedderPolicy: 'unsafe-none' | 'require-corp' | 'credentialless' | false;
```

Comment:
suggestion:

Could you also please add this new value to the types in the module? :)

https://github.com/Baroshem/nuxt-security/blob/abc860d50b323c84583eaa9c1f270f97808b1cdd/src/types/headers.ts#L5

Reply:
Nice, forgot that, done :)